community/kernel/src/main/java/org/neo4j/kernel/impl/spi/KernelContext.java

```diff
@@ -19,13 +19,16 @@
  */
 package org.neo4j.kernel.impl.spi;
 
-import org.neo4j.io.fs.FileSystemAbstraction;
-
 import java.io.File;
 
+import org.neo4j.io.fs.FileSystemAbstraction;
+import org.neo4j.udc.UsageDataKeys;
+
 public interface KernelContext
 {
     FileSystemAbstraction fileSystem();
 
     File storeDir();
+
+    UsageDataKeys.OperationalMode operationalMode();
```

Comment:
Method is unused now but can serve a good purpose replacing `operationalMode` in config.